src/tree.service.ts

```diff
@@ -24,6 +24,8 @@ export class TreeService {
   public nodeExpanded$: Subject<NodeExpandedEvent> = new Subject<NodeExpandedEvent>();
   public nodeCollapsed$: Subject<NodeCollapsedEvent> = new Subject<NodeCollapsedEvent>();
 
+  public controllers: Object = { };
```

Comment:
`any` it is better to [use any](https://stackoverflow.com/questions/18961203/typescript-any-vs-object) in this case